### PR TITLE
Update abstract and other miscellaneous edits before resubmission

### LINKED ISSUE #96: Distill overview
The Distill Update 2018 (http://doi.org/10.23915/distill.00013) has some interesting information about their publishing process.  We currently discuss Distill's focus on interactivity but could also mention their peer review through GitHub issues and use of continuous integration.

### PR DESCRIPTION
**Pull request by agitter**
Closes #96.  This pull request also addresses some items in #97.

I won't touch the paragraphs being edited in #99 until after it is merged, if I need to at all.

We wanted to present deep review early in the introduction, but the intro focuses too much on review manuscripts and not enough on why others should use manubot.  I plan to move parts of the intro to a new discussion sub-section on why collaborative writing is beneficial for review manuscripts.

I also removed my funding.  That grant was relevant for deep review but not Manubot or meta review.

### CODE REVIEW

File: content/01.abstract.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## Abstract {.page_break_before}
 
-Scientific literature reviews are ideal for open, collaborative writing.
-Allowing any interested individual to contribute as an author can strengthen a review, providing broad and fair coverage of the subject matter.
-However, the traditional multi-author writing process breaks down at scale.
-We present techniques for overcoming the challenges of open manuscript writing.
-These include approaches for managing distributed authors and our new software, named Manubot, for automating citation and manuscript building.
+Open, collaborative research is a powerful paradigm that can immensely strengthen the scientific process by integrating broad and diverse expertise.
+However, traditional research and multi-author writing processes break down at scale.
+New tools and workflows that rely on automation can ensure correctness and fairness in massively collaborative research.
```
Comment (dhimmel) [inline content/01.abstract.md:5]: :medal_sports:

**Comment by agitter** [comment]
I reorganized the intro and this is now ready for review.

**Review by cgreene — APPROVED** [review]
I have reviewed and like these changes.

**Comment by agitter** [comment]
@dhimmel would you also like to review before I merge?

File: content/01.abstract.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ## Abstract {.page_break_before}
 
-Scientific literature reviews are ideal for open, collaborative writing.
-Allowing any interested individual to contribute as an author can strengthen a review, providing broad and fair coverage of the subject matter.
-However, the traditional multi-author writing process breaks down at scale.
-We present techniques for overcoming the challenges of open manuscript writing.
-These include approaches for managing distributed authors and our new software, named Manubot, for automating citation and manuscript building.
+Open, collaborative research is a powerful paradigm that can immensely strengthen the scientific process by integrating broad and diverse expertise.
+However, traditional research and multi-author writing processes break down at scale.
```
Comment (slochower) [inline content/01.abstract.md:4]: How do you think Manubot helps with the "research" part?

Comment (slochower) [reply content/01.abstract.md:4]: I could see an argument for using GH issues to discuss relevant papers, like with Deep Review, but that's not really part of Manubot -- the program -- as such.

File: content/02.main-text.md [hunk 3]
```diff
@@ -3,31 +3,21 @@
...
-In addition, we introduce [Manubot](https://github.com/greenelab/manubot-rootstock), the infrastructure we created to enable open manuscript writing online for the Deep Review, which was subsequently adopted by other projects.
+Manuscripts are the cornerstone of scholarly communication, but drafting and publishing manuscripts has traditionally relied on proprietary or offline tools that do not support _open scholarly writing_, in which anyone is able to contribute and the contribution history is preserved and public.
+We introduce [Manubot](https://github.com/greenelab/manubot-rootstock), a new tool and infrastructure for authoring scholarly manuscripts in the open, and report how it was instrumental for the collaborative project that led to its creation.
 
+Based on our experience leading a recent open review [@tag:techblog_manubot], we discuss the advantages and challenges of open collaborative writing, a form of crowdsourcing [@pmcid:PMC4719068].
+Our review manuscript [@doi:10.1098/rsif.2017.0387] was code-named the Deep Review and surveyed deep learning's role in biology and precision medicine, a research area undergoing explosive growth.
```
Comment (slochower) [inline content/02.main-text.md:10]: I don't personally have a problem with the language, but I wasn't an author on Deep Review, and a scrupulous person could object to using "our." I don't suggest any changes, but I just thought I'd mention it.

File: content/02.main-text.md [hunk 4]
```diff
@@ -3,31 +3,21 @@
...
 We initiated the Deep Review by creating a GitHub repository (<https://github.com/greenelab/deep-review>) to coordinate and manage contributions.
 GitHub is a platform designed for collaborative software development that is adaptable for collaborative writing.
-From the start, we made the GitHub repository public, applying a [Creative Commons Attribution License](https://github.com/greenelab/deep-review/blob/master/LICENSE.md) to the manuscript.
-Next, we encouraged anyone interested to contribute by proposing changes or additions.
+From the start, we made the GitHub repository public with a [Creative Commons Attribution License](https://github.com/greenelab/deep-review/blob/master/LICENSE.md).
```
Comment (dhimmel) [inline content/02.main-text.md]: "under" may be better than "with", but both work.

File: content/01.abstract.md (hunk 2 above)
Comment (dhimmel) [reply content/01.abstract.md:4]: I understood this in the broad sense. Projects that do "open, collaborative research" need a way to write their papers. Additionally, many of these projects can be ongoing and at the "paper" stage for a long time. Therefore Manubot doesn't need to be a complete solution for all open, collaborative research needs, but is an important part of the manuscript preparation stage. @agitter was that what you were thinking?

At a larger scale, where research applies to a whole field, the more teams using manubot, the more open and collaborative the field will be.

**Review by dhimmel — APPROVED** [review]
Fantastic edits!

File: content/02.main-text.md (hunk 4 above)
Comment (vsmalladi) [reply content/02.main-text.md]: I would go with "under" seems to flow better.

**Review by vsmalladi — APPROVED** [review]
I like the updates. Looks good/

File: content/01.abstract.md (hunk 2 above)
Comment (agitter) [reply content/01.abstract.md:4]: @slochower my reasoning was along the lines of what @dhimmel said.  For open projects, e.g. https://think-lab.github.io/p/rephetio/#realtime-open-science-thinklab, "traditional" practices (for some definition of traditional) no longer work well.  Having a distributed forum for discussion and synchronizing data, code, and results is necessary.

Manubot does not directly address that, but it is a component of an open distributed research workflow.  Analysis may be coordinated in one repository and writing in another, but I believe that using Manubot for writing will help enable this type of collaborative research.  Contributors could be incentivized by knowing that their work would be fairly credited and they would have the opportunity to write the research product and be recognized as authors.

If that is not coming through clearly or is an overreach given Manubot's current functionality, we can consider alternative phrasing.

File: content/02.main-text.md (hunk 3 above)
Comment (agitter) [reply content/02.main-text.md:10]: I couldn't find specific examples, but I believe I have seen "our" used previously in the literature even when it only pertained to a subset of the authors.  An alternative is to be more specific:
> Based on our experience (DSH, CSG, AG) leading...

but that seems awkward to me.  I'm inclined to leave it as is unless any of us have a problem with this.

Having an author contribution statement could also help clarify which of us were involved in the Deep Review.  Does anyone want to add that?

If we do change these sentences, we'll have to propagate the updates to the other paragraphs that pertain to Deep Review as well.

File: content/02.main-text.md (hunk 4 above)
Comment (agitter) [reply content/02.main-text.md]: Fixed in 1aa6a62

File: content/02.main-text.md (hunk 3 above)
Comment (slochower) [reply content/02.main-text.md:10]: Leave as is, is fine with me.

File: content/01.abstract.md (hunk 2 above)
Comment (slochower) [reply content/01.abstract.md:4]: @dhimmel @agitter Okay, I see your points. I guess I bristled at the strong language. Certainly there have been many large, multi-author projects that worked before Manubot. They are definitely not ideal. They usually suck in multiple ways. But it can be done. It's just inelegant and error-prone. Having said that, I'm not sure how I would rephrase things. Maybe instead of saying "break down at scale" we could say "become very challenging to manage at scale" or something like that. Thoughts?

Also, @agitter, you mentioned:
> Analysis may be coordinated in one repository and writing in another, 

This is a good point. I tried putting Manubot inside another repository once, to bundle code and the writing, and so I could use relative paths to figures so, e.g., `../figure.png` would always pull the latest `figure.png` saved by the code. But that required changing a bunch of little things, so it seems easiest to keep the writing and analysis in separate repositories. We can revisit this in a Manubot Issue if it is necessary.

Comment (agitter) [reply content/01.abstract.md:4]: @slochower would it be better to remove "multi-author" and say

> However, traditional research and writing processes break down at scale.

That leaves "traditional" more open to interpretation.  If "traditional" means passing around a Word document with comments or LaTeX files, that does not scale well and is feasible but quite painful with many authors.

I don't want the abstract to be bombastic, but I do like the phrase "break down at scale" because I find it central to the need for Manubot.

Comment (slochower) [reply content/01.abstract.md:4]: I don't think that is an improvement, "multi-author" is a key point. I'm okay to leaving this sentence as it is. I forgot that it was the abstract, and I do like short, declarative sentences in the abstract.

Comment (agitter) [reply content/01.abstract.md:4]: Okay.  This isn't perfect, but I'll merge these changes.  Please feel welcome to rewrite this sentence in a new pull request if you come up with better phrasing.

**Comment by agitter** [comment]
@dhimmel I'm not sure why the [build](https://travis-ci.org/greenelab/meta-review/builds/439587623) is failing after I merged this
```
Error fetching metadata for doi:10.6084/m9.figshare.5346577.
Invalid response from https://data.datacite.org/10.6084%2Fm9.figshare.5346577:
The resource you are looking for doesn't exist.
```

That URL works fine in a web browser.

**Comment by dhimmel** [comment]
The DOI Content Negotiation hits a different resource/server than the URL resolution I believe. Currently, I get the same error when I try:

```
manubot cite doi:10.6084/m9.figshare.5346577
```

We've had this issue in the past: see https://github.com/datacite/datacite/issues/187, and eventually it will get fixed upstream or resolve itself. Tagging @mfenner who won't be able to get to this immediately because he's leading a [workshop at FORCE2018](https://force2018.sched.com/event/GKCd) (which I hope to say stop by soon to say hi)!

@agitter let's wait a little, hopefully it fixes itself.

**Comment by mfenner** [comment]
Content negotiation requests for DataCite DOIs are forwarded from doi.org to data.datacite.org. We had an issue with our Solr Search index (needed by DOI content negotiation) early this morning that was resolved at 3:46 AM EDT this morning. You still have issues with content negotiation for this DOI?

**Comment by dhimmel** [comment]
> You still have issues with content negotiation for this DOI?

As of 2018-10-10 13:22 Montreal time, yes:

```sh
curl --location --silent \
  --header "Accept: application/vnd.citationstyles.csl+json" \
  https://doi.org/10.6084/m9.figshare.5346577
```

producing `The resource you are looking for doesn't exist.`

Will keep trying every few hours and let you know if the issue resolves! Thanks for the quick input, I know a lot of the team is at FORCE, has been nice meeting everyone.

**Comment by agitter** [comment]
This has been resolved for us now.  I also restarted the Travis build and deployed the new version of the manuscript.
